docs/en/ingest-management/elastic-agent/configuration/examples/config-file-example-apache.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[config-file-example-apache]]
+= Config file example: Apache HTTP Server
+
+++++
+<titleabbrev>Apache HTTP Server</titleabbrev>
+++++
+
+Include these sample settings in your standalone {agent} `elastic-agent.yml` configuration file to ingest data from Apache HTTP server.
+
+* <<config-file-example-apache-logs>>
+* <<config-file-example-apache-metrics>>
+
+[[config-file-example-apache-logs]]
+== Apache HTTP Server logs
+
+["source","yaml"]
+----
+outputs: <1>
+  default:
+    type: elasticsearch <2>
+    hosts:
+      - '{elasticsearch-host-url}' <3>
+    username: '${ES_USERNAME}'
+    password: '${ES_PASSWORD}'
+agent:
+  download: <4>
+    sourceURI: 'https://artifacts.elastic.co/downloads/'
+  monitoring: <5>
+    enabled: true
+    use_output: default
+    namespace: default
+    logs: true
+    metrics: true
+inputs: <6>
+  - id: {user-defined-unique-id} <7>
+    name: apache-1
+    type: logfile <8>
+    use_output: default
+    data_stream: <9>
+      namespace: default
+    streams:
+      - id: {user-defined-unique-id} <7>
+        data_stream:
+          dataset: apache.access <10>
+          type: logs
+        paths: <11>
+          - /var/log/apache2/access.log*
+          - /var/log/apache2/other_vhosts_access.log*
+          - /var/log/httpd/access_log*
+        tags:
+          - apache-access
+        exclude_files:
+          - .gz$
+      - id: {user-defined-unique-id} <7>
```

Comment:
This ID needs to be different from the one above on line 42.

Add the data_stream dataset as a suffix is a good way to do this, so this could be `{user-defined-unique-id}-apache.error` here and `{user-defined-unique-id}-apache.access` above. Anything that results in distinct IDs works, it doesn't have to be this.

Reply:
Nice catch @cmacknz!

I've updated both the logs and the metrics examples. Here's the logs one:

![screenshot](https://github.com/elastic/ingest-docs/assets/41695641/75b438c3-97b5-43b4-9544-0d9f1c49b1c0)

...

![cap pn](https://github.com/elastic/ingest-docs/assets/41695641/fcb0623b-5b27-438e-8b1b-d4ea370f9882)

Reply:
I wonder if we should just explain it, something like "insert a unique identifier here"

then provide the stylistic recommendation for the user to do {user-defined-unique-id}-apache.error

Reply:
@nimarezainia  I like it!

For the logging example I've changed the code example to:  

`id: "insert a unique identifier here" <10>`

And the description to: 
> <10> Specify a unique ID for each individual input stream. Naming the ID by appending the associated `data_stream` dataset (for example `{user-defined-unique-id}-apache.access` or `{user-defined-unique-id}-apache.error`) is a recommended practice, but any unique ID will work.

For the metrics example I've changed the code example to:

`id: "insert a unique identifier here" <9>`

And the description to:
> <9> Specify a unique ID for each individual input stream. Naming the ID by appending the associated `data_stream` dataset (for example `{user-defined-unique-id}-apache.status`) is a recommended practice, but any unique ID will work.